maintainers/scripts/update.py

```diff
@@ -24,15 +25,16 @@ async def check_subprocess(*args, **kwargs):
     Emulate check argument of subprocess.run function.
```

Comment:
There is still the comment and function name missing from https://github.com/NixOS/nixpkgs/pull/194908/files#r990783424

Reply:
I just cherry-picked the other commit.  I didn't realise you were looking for it to be identical.

Reply:
Sorry, I should have been clearer. I was not looking for them to be identical, just so that the review comments from the other PR are addressed. Cherry-picking works for me, though.